### PR TITLE
Ticket 8

### PR DESCRIPTION
**Pull request by omar-baggio**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -252,3 +252,42 @@ describe("ERROR: POST /api/articles/:article_id/comments", () => {
...
+      .patch(`/api/articles/${article_id}`)
+      .send({
+        inc_votes: 10,
+      })
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Not Found");
+      });
+  });
+});
```
Comment (Jodiech) [inline __tests__/app.test.js:316]: Please add a test for the following:

1. 201 that will ignore any additional properties 👍 
2. 400, missing required field(s), e.g. no username or body properties
3. 404, username does not exist

File: __tests__/app.test.js [hunk 2]
```diff
@@ -252,3 +252,42 @@ describe("ERROR: POST /api/articles/:article_id/comments", () => {
...
+describe("ERROR: Patch /api/articles/:article_id", () => {
+  test("400: responds with an error message when passed a bad request", () => {
+    const article_id = "invalid_type";
+    return request(app)
+      .patch(`/api/articles/${article_id}`)
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Bad Request");
+      });
+  });
```
Comment (Jodiech) [inline __tests__/app.test.js:292]: Interesting that you have chosen to write your test by declaring an article_id however. You may have done this with previous tickets, so no need to change this if it's consistent. It just makes your test suite quite difficult to read.

**Review by Jodiech — COMMENTED** [review]
Hey 👋 nice work on this ticket! Please add some more tests (see comments below) I'd like you to resubmit when you're done 👍

**Review by lewispricey — APPROVED** [review]
Great work Omar, thanks for implementing the extra tests Jodie suggested 

Merge away when you're ready 🚀

**Pull request merged**